### ISSUE TITLE
prerender error

### ISSUE THREAD

**Issue by Vibes-INS** [brillout/vite-plugin-ssr#572]
### Description

repository github that bug: https://github.com/Vibes-INS/vite-react-isr-demo/tree/chore/isr

My code can be compiled normally without prerender:
```ts
export default defineConfig({
  plugins: [
    react(),
    ssr(),
    vercel(),
    vercelSsr(),
  ],
  vercel: {},
  server: {
    port: 3000,
  },
})
```

But, if i add `prerender`:
```ts
ssr({
  prerender: {
    noExtraDir: true,
  },
})
```

### Error Message + Error Stack

```shell
vite-plugin-ssr v0.4.64 pre-rendering HTML...
Error: [vite-plugin-ssr@0.4.64][Bug] You stumbled upon a bug in vite-plugin-ssr's source code. Reach out at https://github.com/brillout/vite-plugin-ssr/issues/new or https://discord.com/invite/qTq92FQzKb and include this error stack (the error stack is usually enough to fix the problem). A maintainer will fix the bug (usually under 24 hours). Don't hesitate to reach out as it makes vite-plugin-ssr more robust. 
    at setGlobalContextViteConfig (/Users/coder/Code/vite-react-isr-demo/node_modules/.pnpm/vite-plugin-ssr@0.4.64_vite@3.0.5/node_modules/vite-plugin-ssr/dist/cjs/node/globalContext.js:22:24)
    at Object.configResolved (/Users/coder/Code/vite-react-isr-demo/node_modules/.pnpm/vite-plugin-ssr@0.4.64_vite@3.0.5/node_modules/vite-plugin-ssr/dist/cjs/node/plugin/plugins/setGlobalContext.js:12:60)
    at file:///Users/coder/Code/vite-react-isr-demo/node_modules/.pnpm/vite@3.0.5/node_modules/vite/dist/node/chunks/dep-5cb728cb.js:62736:39
    at Array.map (<anonymous>)
    at Module.resolveConfig (file:///Users/coder/Code/vite-react-isr-demo/node_modules/.pnpm/vite@3.0.5/node_modules/vite/dist/node/chunks/dep-5cb728cb.js:62736:10)
    at async runPrerender (/Users/coder/Code/vite-react-isr-demo/node_modules/.pnpm/vite-plugin-ssr@0.4.64_vite@3.0.5/node_modules/vite-plugin-ssr/dist/cjs/node/prerender/runPrerender.js:30:24)
    at async prerender (/Users/coder/Code/vite-react-isr-demo/node_modules/.pnpm/vite-plugin-ssr@0.4.64_vite@3.0.5/node_modules/vite-plugin-ssr/dist/cjs/node/prerender.js:6:5)
    at async triggerFullBuild (/Users/coder/Code/vite-react-isr-demo/node_modules/.pnpm/vite-plugin-ssr@0.4.64_vite@3.0.5/node_modules/vite-plugin-ssr/dist/cjs/node/plugin/plugins/autoFullBuild.js:58:9)
    at async Object.writeBundle (/Users/coder/Code/vite-react-isr-demo/node_modules/.pnpm/vite-plugin-ssr@0.4.64_vite@3.0.5/node_modules/vite-plugin-ssr/dist/cjs/node/plugin/plugins/autoFullBuild.js:24:17)
    at async Promise.all (index 0)
    at async PluginDriver.hookParallel (file:///Users/coder/Code/vite-react-isr-demo/node_modules/.pnpm/rollup@2.79.1/node_modules/rollup/dist/es/shared/rollup.js:22676:9)
    at async file:///Users/coder/Code/vite-react-isr-demo/node_modules/.pnpm/rollup@2.79.1/node_modules/rollup/dist/es/shared/rollup.js:23804:13
    at async catchUnfinishedHookActions (file:///Users/coder/Code/vite-react-isr-demo/node_modules/.pnpm/rollup@2.79.1/node_modules/rollup/dist/es/shared/rollup.js:23126:20)
    at async doBuild (file:///Users/coder/Code/vite-react-isr-demo/node_modules/.pnpm/vite@3.0.5/node_modules/vite/dist/node/chunks/dep-5cb728cb.js:43580:20)
    at async build (file:///Users/coder/Code/vite-react-isr-demo/node_modules/.pnpm/vite@3.0.5/node_modules/vite/dist/node/chunks/dep-5cb728cb.js:43403:16)
    at async CAC.<anonymous> (file:///Users/coder/Code/vite-react-isr-demo/node_modules/.pnpm/vite@3.0.5/node_modules/vite/dist/node/cli.js:747:9)
```

**Comment by brillout** [comment]
Fix released in `0.4.65`.

**Comment by brillout** [comment]
Btw. in case your company is up for it: https://github.com/sponsors/brillout.
